src/coreclr/jit/llvm.cpp

```diff
@@ -1656,25 +1685,32 @@ void Llvm::storeLocalVar(GenTreeLclVar* lclVar)
 {
     if (lclVar->gtFlags & GTF_VAR_DEF)
```

Comment:
Hmm, while you're at it, delete this `lclVar->gtFlags & GTF_VAR_DEF` check, it's not useful (`STORE_LCL_VAR` should always have this flag set).

Reply:
Done